tests/api/test_initial_data.py

```diff
@@ -228,6 +237,39 @@ def test_create_project_summaries():
     assert migrated_project_summary.name == project.metadata.name
 
 
+def test_align_schedule_labels():
```

Comment:
please add some more schedules with edge cases. we need to make sure this will not break in field